README.md

```diff
@@ -16,18 +22,28 @@ Then add **hubot-help** to your `external-scripts.json`:
 ["hubot-help"]
 ```
 
-## Configuration
+Configuration
+-----------------
 
 You can set various environment variables to tune up the behavior of thios help plugin:
 
 - `HUBOT_HELP_REPLY_IN_PRIVATE` (set to any value) will force calls to `hubot help` to be answered in private
 - `HUBOT_HELP_DISABLE_HTTP` (set to any value) will disable the web interface for help
 - `HUBOT_HELP_HIDDEN_COMMANDS` comma-separated list of commands that will not be displayed in help
 
-## Sample Interaction
+Development
+-----------------
+
+For tests:
+
+  npm test
```

Comment:
You either need 4 spaces or wrap npm test with 3 ticks. Two spaces is not formatting `npm test` as a code block